docs/src/components/Tabs.tsx

```diff
@@ -83,12 +72,15 @@ interface Props {
   activeTab: string;
 }
 
-const Tabs = ({ activeTab, tabs }: Props) => (
-  <Box padding={{ left: "medium" }} margin={{ top: "none" }}>
-    {tabs.map(tab => (
-      <Tab key={tab.title} tab={tab} isActive={activeTab === tab.slug} />
-    ))}
-  </Box>
-);
+const Tabs = ({ activeTab, tabs }: Props) => {
+  if (tabs.length === 0) return null;
+  return (
+    <Container>
+      {tabs.map(tab => (
+        <Tab key={tab.title} tab={tab} isActive={activeTab === tab.slug} />
+      ))}
+    </Container>
+  );
+};
```

Comment:
These changes attempt to make the tab shadow trick more stable, the prior implementation was interfering with examples.